### DISCUSSION TITLE
Is it still maintained?

### DISCUSSION THREAD

**Discussion by JasonMarechal25** [open-source-parsers/jsoncpp#1544] [General]
Hello
Given the last commit is dated from Jun 27, 2023 and that several PR are opened but neither denied nor merged, I'm wondering if this repository is still maintained?

**Comment by baylesj** [comment]
Hello, 

This is a great question. I know me and a few other maintainers, like @BillyDonahue, still have interest in this repository but there hasn't been much traction this past year.

I have been working on getting things into a better place, including deleting the TravisCI integration that hasn't been working, and starting to clear the backlog of pull requests that have piled up.

I think the repository is now in a better place and hopefully we can start to get some more traction towards a new release and more consistent updates.

**Reply by baylesj** [reply]
At this point, we have working Meson and CMake builds for Mac OS, Windows, and Linux, as well as code coverage for all pull requests.

**Reply by BillyDonahue** [reply]
I haven't been able to move without Travis, and I didn't have the access to fix it, so it's been frustrating.  Thanks for picking up the challenge, Jordan.

**Reply by baylesj** [reply]
The new build system using Actions is a lot faster than Travis + Appveyor anyway, so it does feel like a win to me. And we still have CMake builders plus new Meson builders and code coverage, so I feel like things are in a pretty decent state.

**Reply by baylesj** [reply]
Happy to unblock you, @BillyDonahue. Definitely appreciate all the work you have put in on the repository.

**Reply by bcsgh** [reply]
Looks like the PR's are getting back logged again?

**Reply by baylesj** [reply]
I attempt to circle back to this repository whenever I have spare cycles for it, which unfortunately does mean that there are periods of activity where PRs will get backlogged. I have attempted to clean things up a little bit recently so hopefully we are in a better place + a 1.9.7 bugfix release.
